kpi/008-links-health.adoc

```diff
@@ -2,41 +2,39 @@
 
 === Measurement
 
-The number of broken links in each individual metadata record.  Broken links
-include links which, when accessed, result in a 4xx or 5xx HTTP error.footnote:[https://httpstatuses.com]
-
-Also being measured is the use of HTTPS (with a valid SSL certificate) as the
-link protocol throughout WIS Metadata.
+Links are valid (no 4xx or 5xx HTTP status errors) and available through the HTTPS protocol. 
 
 === Rationale for measurement
 
 Broken links damage the user experience and gives the impression to users that
-a website is not maintained (88% of the online consumers are less likely to
-return to a site after a bad experience.footnote:[https://review42.com/web-design-statistics])
-In addition, having numerous broken links affects the reputation and rank of
-your website when indexed by mass market search engines.
+a website is not maintained. 
 
-HTTPS is increasingly becoming a requirement for numerous agencies as well as the
-suggested protocol vs. HTTP.  Having non-HTTPS links in a WCMP document often leads to mixed
-content errors in web applications deployed via HTTPS for example, and using AJAX/XHR design
-patterns.  HTTPS supports secure, authoritative and trustworthy links as part of WIS Metadata.
+HTTPS is increasingly becoming a requirement for numerous agencies.  Metadata records with non-HTTPS links often leads to mixed
+content errors in web applications deployed via HTTPS. HTTPS supports secure, authoritative and trustworthy links as part of WIS metadata.
 
 === Rules
 
 .Links health implementation rules
+
 |===
 |Rule |Score
 
-|Each valid link present
+|8.1: The link resolves, when it is present in `URL` element, `fileName` element, `href` attribute, or `codeList` attribute. 
```

Comment:
Should we prefix the elements with a namespace prefix (i..e `gmd:URL`, `gmd:fileName`)?

Reply:
I was wondering this too, but I didn't do this with any of the  element "names" in other KPIs so I decided to keep it this way for consistency. BUT if you think it's a better practice to include the namespace in all KPIs, then I'm ok with that too.

Reply:
As discussed, if we specify element names, then they should be prefixed explicitly, for clarity.